Makefile

```diff
@@ -153,23 +153,34 @@ php_proto:
 DOCKER_IMAGES_FOR_TEST = mariadb mysql56 mysql57 percona percona57
 DOCKER_IMAGES = common $(DOCKER_IMAGES_FOR_TEST)
 docker_bootstrap:
-	for i in $(DOCKER_IMAGES); do echo "image: $$i"; docker/bootstrap/build.sh $$i || exit 1; done
+	for i in $(DOCKER_IMAGES); do echo "building bootstrap image: $$i"; docker/bootstrap/build.sh $$i || exit 1; done
 
 docker_bootstrap_test:
 	flavors='$(DOCKER_IMAGES_FOR_TEST)' && ./test.go -pull=false -parallel=4 -flavor=$${flavors// /,}
 
 docker_bootstrap_push:
-	for i in $(DOCKER_IMAGES); do echo "image: $$i"; docker push vitess/bootstrap:$$i || exit 1; done
+	for i in $(DOCKER_IMAGES); do echo "pushing boostrap image: $$i"; docker push vitess/bootstrap:$$i || exit 1; done
 
+# Use this target to update the local copy of your images with the one on Dockerhub.
+docker_bootstrap_pull:
+	for i in $(DOCKER_BOOTSTRAP_IMAGES); do echo "pulling bootstrap image: $$i"; docker pull vitess/bootstrap:$$i || exit 1; done
+
+# Note: The default base and lite images (tag "latest") use MySQL 5.7.
+# Images with other MySQL/MariaDB versions get their own tag e.g. "mariadb".
+# We never push the non-"latest" tags though and only provide them for convenience for users who want to run something else than MySQL 5.7.
 docker_base:
-	# Fix permissions before copying files, to avoid AUFS bug.
+# Fix permissions before copying files, to avoid AUFS bug.
```

Comment:
In the past I got burned by it but I don't remember why.

By default, it would also get echoed to stdout. But since we have `MAKEFLAGS = -s` at the top of the file, it really makes no difference that bash is executing the command.

Given all this I've reverted the change. Thanks for calling it out.